packages/storage/src/indexed-db/index.ts

```diff
@@ -458,12 +458,14 @@ export class IndexedDb implements IndexedDbInterface {
     });
   }
 
-  async addEpoch(startHeight: bigint, index?: bigint): Promise<void> {
```

Comment:
We're never actually using the `index` argument anywhere, so I just got rid of it.